integrations/prisma-exception/lint-staged.config.js

```
@@ -7,27 +7,31 @@
  * {@link https://github.com/belgattitude/nextjs-monorepo-example/blob/main/docs/about-lint-staged.md}
  */
 
-const {
-  concatFilesForPrettier,
-  getEslintFixCmd,
-} = require('../../lint-staged.common.cjs');
+import path from 'node:path';
+import url from 'node:url';
+
+import {
+    concatFilesForPrettier,
```

ESLint

[error] Delete `··`

(prettier/prettier)